packages/wonder-blocks-tabs/src/components/__tests__/tab-panel.test.tsx

```diff
@@ -125,5 +146,59 @@ describe("TabPanel", () => {
                 );
             });
         });
+
+        describe("Focus", () => {
+            it("should be focusable if there are no focusable elements in the panel", async () => {
+                // Arrange
+                render(
+                    <TabPanel {...props} active={true}>
+                        No focusable elements
+                    </TabPanel>,
+                );
+                const tabPanel = await screen.findByRole("tabpanel");
+
+                // Act
+                await userEvent.tab();
+
+                // Assert
+                expect(tabPanel).toHaveFocus();
+            });
+
+            it.each([
+                {
+                    element: <a href="#link">Link Example</a>,
+                    label: "Link",
+                },
+                {
+                    element: <input type="text" />,
+                    label: "Input",
+                },
+                {
+                    element: <button>Button</button>,
+                    label: "Button",
+                },
+                {
+                    element: <textarea />,
+                    label: "Textarea",
+                },
+            ])(
+                "should not be focusable if there is a focusable element in the panel ($label)",
+                async ({element}) => {
+                    // Arrange
+                    render(
+                        <TabPanel {...props} active={true}>
+                            {element}
+                        </TabPanel>,
+                    );
+                    const tabPanel = await screen.findByRole("tabpanel");
+
+                    // Act
+                    await userEvent.tab();
+
+                    // Assert
+                    expect(tabPanel).not.toHaveFocus();
```

Comment:
Should this assert which element does have focus, to ensure the contents are focusable? Maybe in a separate set of tests?